pkgs/tools/security/gnupg/24.nix

```diff
@@ -35,6 +35,12 @@ stdenv.mkDerivation rec {
     ./24-allow-import-of-previously-known-keys-even-without-UI.patch
     # Patch for DoS vuln from https://seclists.org/oss-sec/2022/q3/27
     ./v3-0001-Disallow-compressed-signatures-and-certificates.patch
+    # Patch for emacs easypg https://dev.gnupg.org/T6481
+    # It's merged to gnupg master so can be removed in the next major version
+    (fetchpatch {
+      url = "https://files.gnupg.net/file/data/iapdg4omgofpqs5c6gd5/PHID-FILE-pym22ajak2knrci6sc2h/file";
```

Comment:
Could we use this url rather `https://dev.gnupg.org/rG2f872fa68c6576724b9dabee9fb0844266f55d0d?diff=1` ? This should redirect to the same content , so the sha256 sum can stay the same. But this url allows to trace this back to a commit ( by removing the `?diff=1`) and it does not look like some random attachment that somebody might have posted to a bugtracker.

Reply:
Sure.